completions/fish/meson.build

```diff
@@ -0,0 +1,15 @@
+fish_completion_dir = get_option('fishcompletiondir')
+fish = dependency('fish', required : false)
+
+if fish_completion_dir == '' and fish.found()
+    fish_completion_dir = fish.get_pkgconfig_variable('completionsdir')
+elif fish_completion_dir == ''
+    # Fish does not look in /usr/local/ so get_option("datadir") is not used
+    # to avoid breaking when --prefix is not usr
```

Comment:
Can we put this comment another way? Say,

"Fish does not look in /usr/local/, which is the default prefix, so we cannot use get_option("datadir"). Instead, they recommend custom completions to be installed into /usr/share/fish/vendor_completions.d, so we use that. Note that this is a fallback in case you neither have fish installed, nor have specified a path explicitly."